README.md

```diff
@@ -461,6 +461,26 @@ If the implementation package uses ESM and specifies `"type": "module"`, then yo
 
 This also applies if the implementation package has `exports` in its package.json.
 
+##### Peer dependencies
+
+Definitely Typed allows `peerDependencies` in `package.json`.
+Peer dependencies can help prevent situations where a package manager unexpectedly installs too-new versions or more than one version of the same package.
+However, peer dependencies have downsides; package managers differ in their handling of peer dependencies (e.g., `yarn` does not auto-install them, `npm` requires `--legacy-peer-deps` for mismatches).
+As such, PRs introducing new peer dependencies require maintainer approval and should be limited to specific circumstances.
+
+In general, types packages should only have a peer dependency if the upstream package has a peer dependency on the same package (or its types).
```

Comment:
If there's a way to call this out, I think this is the one-line takeaway of the whole new section.

Reply:
Did a little bolding.